conjur-oss/values.schema.json

```diff
@@ -0,0 +1,234 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "required": [
+    "database",
+    "dataKey",
+    "ssl"
+  ],
+  "properties": {
+    "account": {
+      "type": "string"
+    },
+    "affinity": {
+      "type": "object"
+    },
+    "authenticators": {
+      "type": "string"
+    },
+    "conjurLabels": {
+      "type": "object"
+    },
+    "database": {
+      "required": [
+        "ssl"
+      ],
+      "properties": {
+        "password": {
+          "type": "string"
+        },
+        "ssl": {
+          "required": [
```

Comment:
The `ssl.key` value should only be required if `ssl.cert` is set. I think you can use the `dependencies` keyword (see https://stackoverflow.com/questions/38717933/jsonschema-attribute-conditionally-required); e.g. add something like this (not sure if I have the syntax correct):
```
"dependencies": {
  "cert": { "required": ["key"] }
}
```
at line 45.